requirements.txt

```diff
@@ -29,6 +29,7 @@ sentry-sdk==1.9.8
 simplejson==3.17.6
 structlog==22.1.0
 structlog-sentry==2.0.0
+sql-metadata==2.6.0
```

Comment:
Was this added to the https://github.com/getsentry/pypi/ repo as well?

Reply:
Yep